### ISSUE TITLE
Cannot init DB on MySQL (sch_usr_notification)

### ISSUE THREAD

**Issue by jakubklimek** [UnifiedViews/Core#425]
When executing data-core.sql
The insert command does not match the table schema, which has 6 columns.
```
0	168	21:21:03	INSERT INTO `sch_usr_notification` VALUES (NULL,1,1,1),(NULL,2,1,1)	Error Code: 1136. Column count doesn't match value count at row 1	0.031 sec
```

**Comment by jakubklimek** [comment]
Works when I add 1,1 and I presume that is the correct setting

**Comment by skrchnavy** [comment]
@eea03 is this related to #419 ?

**Comment by eea03** [comment]
Seems this was caused by incorrect merge of #419 and changes in develop merged by #417. 
Fixed, tested.

**Comment by tomas-knap** [comment]
@jakubklimek Please check it is ok now
